apps/frontend/src/components/meetings/InitialMeet.tsx

```
@@ -0,0 +1,46 @@
+  const { session } = useAuth()
+  const [loading, setLoading] = useState(true)
+  const router = useRouter()
+  const fetchLatestMeet = useMeetsStore(
+    (state: MeetsStoreType) => state.fetchLatestMeet
+  )
+
+  useEffect(() => {
+    const getMeetId = async () => {
+      try {
+        const meet: Meet | null = await fetchLatestMeet(session)
+        if (meet && meet._id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)